src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,7 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return $property->getMinOccurs() > 1 || !is_null($property->getMaxOccurs());
```

Comment:
Probably the safest solution would be
maxOccurs != null || minOccurs > 1 || minOccurs < maxOccurs
we should consider valid all these cases:

```
<property name="name" type="text_line" minOccurs="1" maxOccurs="20">

<property name="name" type="text_line" maxOccurs="20">

<property name="name" type="text_line" minOccurs="2">

<property name="name" type="text_line" minOccurs="0" maxOccurs="20">
```